hello.js

```diff
@@ -0,0 +1,34 @@
+// Gets user input
+/*var prompt = require('syncprompt');
+var name = prompt("What is your name? ");
+
+// Uses  to print out information
+function printHelloName(){
+  for (var i = 0; i < 5; i++) {
+      console.log("Hello " + name);
+    }
+  }  
+printHelloName();
+*/
+
+// With paramater name
+/*function printHelloName(name){
+  name = "Milica";
+  for (var i = 0; i < 5; i++) {
+      console.log("Hello " + name);
+    }
+  }  
+printHelloName();
+*/
+
+//Command Line Argument
+function printHelloName(name) {
+  if (process.argv[2] != '' || process.argv[2] != isNaN) {
```

Comment:
does this works in your local? Because when I call "node hello.js" I just get:
`Hello undefined` 5x :/ 
Try changing the if a little bit, and let me know if you need help :)